app/controllers/api/v1/users_controller.rb

```diff
@@ -3,7 +3,7 @@ module V1
     class UsersController < ApplicationController
       def create
         user = User.create! user_params
-        represent_response UserPresenter.new(user)
+        represent_response UserPresenter.new(user), Settings.http_code.code_201
```

Comment:
set default r nên k cần pass explicit status code nữa nhể :v?
```suggestion
        represent_response UserPresenter.new(user)
```